store/src/java/com/zimbra/cs/mailclient/MailConnection.java

```diff
@@ -71,7 +73,7 @@ protected MailConnection(MailConfig config) {
      *
      * @throws IOException if an I/O error occurs
      */
-    public synchronized void connect() throws IOException {
+    public synchronized void connect() throws IOException, CertificateException {
```

Comment:
This doesn't build.  Callers need modification to cope with CertificationException.